src/org/parosproxy/paros/extension/state/ExtensionState.java

```diff
@@ -74,70 +66,14 @@ public void hook(ExtensionHook extensionHook) {
 	    // ZAP: Changed to check if there is a view.
 	    if (getView() != null) {
 		    extensionHook.getHookMenu().addEditMenuItem(extensionHook.getHookMenu().getMenuSeparator());
-		    extensionHook.getHookMenu().addEditMenuItem(getMenuSessionTrackingEnable());
-		    extensionHook.getHookMenu().addEditMenuItem(getMenuResetSessionState());
 	    }
 	}
 
-	
-	/**
-	 * This method initializes menuViewImage	
-	 * 	
-	 * @return javax.swing.JCheckBoxMenuItem	
-	 */    
-	private JCheckBoxMenuItem getMenuSessionTrackingEnable() {
-		if (menuSessionTrackingEnable == null) {
-		    menuSessionTrackingEnable = new JCheckBoxMenuItem();
-		    menuSessionTrackingEnable.setText(Constant.messages.getString("menu.edit.enableTracking"));	// ZAP: i18n
-			getMenuResetSessionState().setEnabled(menuSessionTrackingEnable.isSelected());
-
-			menuSessionTrackingEnable.addItemListener(new java.awt.event.ItemListener() { 
-
-				@Override
-				public void itemStateChanged(java.awt.event.ItemEvent e) {    
-
-					getModel().getOptionsParam().getConnectionParam().setHttpStateEnabled(menuSessionTrackingEnable.isEnabled());
-					getMenuResetSessionState().setEnabled(menuSessionTrackingEnable.isSelected());
-			        resetSessionState();
-				}
-			});
-
-		}
-		return menuSessionTrackingEnable;
-	}
-	
-
-          //  @jve:decl-index=0:}  //  @jve:decl-index=0:
-	/**
-	 * This method initializes jMenuItem	
-	 * 	
-	 * @return javax.swing.JMenuItem	
-	 */    
-	private ZapMenuItem getMenuResetSessionState() {
-		if (menuResetSessionState == null) {
-			menuResetSessionState = new ZapMenuItem("menu.edit.resetState");

```

Comment:
These (this and the one in the action) messages should be removed, if no longer in use (which don't seem to be).
Removing this option hampers the usability a bit though (but I guess it's fine since the state should not be reset that often).